mysql/covid19.sql

```diff
@@ -48,3 +48,12 @@ CREATE TABLE `notification_message_type` (
   `name` VARCHAR(45) NULL,
   `description` VARCHAR(45) NULL,
   PRIMARY KEY (`id`));
+  
+CREATE TABLE `covid19_db`.`covid_status` (
```

Comment:
database name can be removed as application is connected to only one db

Reply:
ok